apps/core-app/src/pages/DaoOverview.tsx

```diff
@@ -30,6 +43,28 @@ export function DaoOverview() {
             data="May 10, 2022 15:55pm PST"
             info="this is a tooltip"
           />
+          <Box>
```

Comment:
Can you move Box inside the check for dao? Otherwise possible for 3 empty divs to render here.